src/plugins/console/public/application/models/sense_editor/__tests__/integration.test.js

```diff
@@ -91,7 +91,6 @@ describe('Integration', () => {
         '',
         function (err, terms) {
           if (testToRun.assertThrows) {
-            done();
```

Comment:
Hi @patrykkopycinski ! Thanks for working on this, I did not dig into this too deeply (Console's tests are not in a good state) but I recall needing these done callbacks at one state. I am guessing these were failing before we started remove the "done" callback? Would you mind explaining the problem as you understand it (I see CI is failing on these integration tests)

Reply:
Hi @jloleysens, so the issue was that with the new runner we cannot use both `async` and `done` in the same test, so instead of using `async/await` I have used `.then()` to make sure the assertion is run after the editor is updated and seems that solves the issue